### PR TITLE
Local server data updates

### PR DESCRIPTION
**Pull request by Rachael-E**
### Description

This PR updates the data sources for Local Server to the up to date format for tile, map and geoprocessing packages (tpkx, mpkx and gpkx respectively). There are also some associated README updates and the gradle wrappers have all updated too. Gradle files have been updated automatically too, I've matched the autogenerated copyright to the year the sample was created (2017) for consistency. @jenmerritt could you take a look please? The readme checker is throwing an error for "Server" in the title but we can just ignore this and I'll update the script later!

### CODE REVIEW

File: local_server/local-server-services/settings.gradle [hunk 1]
```diff
@@ -4,58 +4,23 @@ rootProject.name = 'local-server-services'
...
-    )
-    ant.unzip(
-            src: file("./f.zip"),
-            dest: file("./samples-data/local_server")
+            src: "https://arcgisruntime.maps.arcgis.com/sharing/rest/content/items/a680362d6a7447e8afe2b1eb85fcde30/data",
+            dest: file("./samples-data/local_server/Contour.gpkx")
     )
-    delete file("./f.zip")
+    
 }
```
Comment (jenmerritt) [inline local_server/local-server-services/settings.gradle]:
```suggestion
}

```

**Review by jenmerritt — APPROVED** [review]
@Rachael-E approving as just a couple of minor comments. But all working when I tested.

- Local Server Feature Layer Readme - line 22: one other `mpkx` change to make?

- In the readme's, not sure if this line should be updated to reflect latest versions or if leaving it as an example:

`To configure the ArcGIS Runtime API for Java v100.10.0 to work with Local Server 100.9.0:`

File: local_server/local-server-services/settings.gradle (hunk 1 above)
Comment (jenmerritt) [reply local_server/local-server-services/settings.gradle]: Diff's are showing no blank line at the end of this file

**Comment by Rachael-E** [comment]
thanks @jenmerritt ! I've made those updates and have updated the Java 100.10.0 version to 100.11.0. The wording would probably have been ok since it's just an example of how to switch local server versions but no harm in updating it.

**Pull request merged**